quix-frontend/client/src/lib/runner/directives/results/console/console-result.ts

```diff
@@ -6,7 +7,27 @@ export default () => {
     restrict: 'E',
     template,
     scope: {
-      query: '<'
-    }
+      query: '<',
+    },
+    link: {
+      async pre(scope: IScope) {
+        
```

Comment:
not sure if newline is justified.

Reply:
Initially it made by prettier, bu in this specific case "link" is an object and expanded object is naturally looks beter with this new line in my opinion :)

Reply:
but if you are about the line 14 so yes, I removed it

Reply:
yes, line 14